internal/session/handle_store.go

```diff
@@ -21,16 +21,16 @@ func (s *Session) handleStore(ctx context.Context, tag string, cmd *proto.Store,
 		defer profiling.Stop(ctx, profiling.CmdTypeStore)
 	}
 
-	if cmd.GetAction().GetSilent() {
+	if cmd.Silent {
 		ctx = contexts.AsSilent(ctx)
 	}
 
-	flags, err := validateStoreFlags(cmd.GetFlags())
+	flags, err := validateStoreFlags(cmd.Flags)
 	if err != nil {
 		return response.Bad(tag).WithError(err), nil
 	}
 
-	if err := mailbox.Store(ctx, cmd.GetSequenceSet(), cmd.GetAction().GetOperation(), flags); errors.Is(err, state.ErrNoSuchMessage) {
+	if err := mailbox.Store(ctx, cmd, flags); errors.Is(err, state.ErrNoSuchMessage) {
```

Comment:
Why leak the command.Store type through to the backend? I liked it more how it was, where Store would just operate on SeqSet and Operation

Reply:
Done